### ISSUE TITLE
rename REAME to README.org

### ISSUE THREAD

**Issue by dmgerman** [scallywag/org-board#12]
Hi Scallywag,

if you rename README to README.org github will still render it in the main page  as it does with the README.

thanks for org-board!

**Comment by scallywag** [comment]
Thanks for the suggestion. While it would look nicer to do that (especially for modern web browsers), I tend to write the documentation in plain text as a comment at the start of org-board.el (which I then copy to the README), so I want to keep the formatting fairly simple.

**Comment by alphapapa** [comment]
Another vote for doing this.  The readme is much harder to read on GitHub than it would be if it were an Org file, which GitHub would render nicely as HTML.

Instead of writing in the commentary and copying into the readme, you could do the opposite: write in README.org, and then export with Org Export to plain-text, which you could copy into the commentary.  Or just keep a minimal commentary, put the bulk of the documentation in the readme and an info page, which would be included in the Emacs package automatically (Org can export directly to info pages).

**Comment by scallywag** [comment]
> Date: Mon, 10 Sep 2018 03:10:24 -0700
> From: alphapapa <notifications@github.com>
> 
> Another vote for doing this. The readme is much harder to read on GitHub than it would be if it were an Org file, which
> GitHub would render nicely as HTML.
> 
> Instead of writing in the commentary and copying into the readme, you could do the opposite: write in README.org, and
> then export with Org Export to plain-text, which you could copy into the commentary. Or just keep a minimal commentary,
> put the bulk of the documentation in the readme and an info page, which would be included in the Emacs package
> automatically (Org can export directly to info pages).

Yes, the README is not great as-is.  I think I will rewrite the doc in
Info, and keep a basic README in Org format.  Thanks for sharing your
thoughts!

**Comment by alphapapa** [comment]
What I mean is, Org can export to info pages, so you can write it in Org.  e.g. this is what I do now in https://github.com/alphapapa/org-super-agenda and it's also how the official Org manual file, `org-manual.org`, is written.

**Comment by scallywag** [comment]
> Date: Tue, 11 Sep 2018 21:59:06 -0700
> From: alphapapa <notifications@github.com>
> Reply-To: scallywag/org-board <reply@reply.github.com>
> Cc: scallywag <charles@aurox.ch>,   Comment <comment@noreply.github.com>
> Content-Type: multipart/alternative;  boundary="--==_mimepart_5b989d1a66841_3a0d3f938a4d45c4115734";  charset=UTF-8
> 
> 
> [1:text/plain Show]
> 
> 
> [2:text/html Hide Save:noname (4kB)]
> 
> What I mean is, Org can export to info pages, so you can write it in Org. e.g. this is what I do now in
> https://github.com/alphapapa/org-super-agenda and it's also how the official Org manual file, org-manual.org, is written.

Aha, I see what you mean.  I will give it a try.
